homeassistant/components/salus/climate.py

```diff
@@ -0,0 +1,110 @@
+"""Climate entity for the Salus integration."""
+
+from homeassistant.components.climate import ClimateEntity
+from homeassistant.components.climate.const import (
+    CURRENT_HVAC_HEAT,
+    CURRENT_HVAC_IDLE,
+    HVAC_MODE_HEAT,
+    HVAC_MODE_OFF,
+    SUPPORT_TARGET_TEMPERATURE,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import ATTR_TEMPERATURE
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN, MANUFACTURER
+
+SUPPORT_FLAGS = SUPPORT_TARGET_TEMPERATURE
+
+
+async def async_setup_entry(
+    hass: HomeAssistant, config_entry: ConfigEntry, async_add_entities
+):
+    """Set up the it500 Salus thermostat."""
+    coordinator, device = hass.data[DOMAIN][config_entry.entry_id]
+
+    temperature_unit = hass.config.units.temperature_unit
+    entity = IT500Salus(coordinator, temperature_unit, device)
+
+    async_add_entities([entity], True)
+
+
+class IT500Salus(CoordinatorEntity, ClimateEntity):
+    """Representation of a Salus IT500 Thermostat."""
+
+    def __init__(self, coordinator, temperature_unit, device):
+        """Initialize the thermostat."""
+        super().__init__(coordinator)
+        self._coordinator = coordinator
+        self._temperature_unit = temperature_unit
+        self._device = device
+
+    @property
+    def name(self):
+        """Return default name for entity."""
+        return f"{self._device.name} Temperature"
+
+    @property
+    def temperature_unit(self):
+        """Return temperature unit user."""
+        return self._temperature_unit
+
```

Comment:
This property could be dropped if you define `self._attr_temperature_unit` inside `__init__` method.
https://github.com/home-assistant/core/blob/1f2a5ae98dccee962195236bae197415c7efb851/homeassistant/components/climate/__init__.py#L205